documentation/book/assembly-metrics.adoc

```diff
@@ -0,0 +1,25 @@
+// This assembly is included in the following assemblies:
```

Comment:
This is not a priority item for Beta so if edits to this will take too long, feel free to postpone this until Post-Beta